client/convert/protobuf.go

```diff
@@ -259,6 +266,15 @@ func CollectionGuaranteeToMessage(g flow.CollectionGuarantee) *entities.Collecti
 	}
 }
 
+func BlockSealToMessage(g flow.BlockSeal) *entities.BlockSeal {
```

Comment:
Can you add tests for these new conversion functions? The format is pretty simple -- see `protobuf_test.go` for examples